media/templates/homepage.mustache

```diff
@@ -80,6 +80,30 @@
 
 <div class="visualclear"></div>
 
+{{#num_pages}}
+<ul class="projectlist pagination">
+{{#previous_page}}
+    <li>
+        <a href="#" aria-label="Previous" data-page="{{.}}" class="page">
+            <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+        </a>
+    </li>
+{{/previous_page}}
+    <li class="description">
+        <a href="#" aria-label="{{current_page}}/{{num_pages}}">
```

Comment:
The link as the top of the page is fixed but this one (at the bottom of the page) still has the link.